database/factories/ModelFactory.php

```diff
@@ -67,7 +67,7 @@
     }
 
     return [
-        'sponsor_id' => $sponsor_id,
+        'sponsor_id' => $sponsor_id, // a random sponsor
         'code' => $faker->ean8, // 8 digit barcode
         'currentstate' => 'requested'
```

Comment:
missed off the trailing , 

I guess stick to one or the other standard
[
1,
2,
3,
] 
or 
[
1,
2,
3
]